src/main/java/org/opentripplanner/updater/bike_rental/GbfsBikeRentalDataSource.java

```diff
@@ -22,6 +23,7 @@
 /**
  * Created by demory on 2017-03-14.
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
At first glance, I don't think this class or `GenericJsonBikeRentalDataSource.java` should need these annotations. The `@JsonIgnoreProperties` annotation is only meant for classes that are serialized/deserialized like the `GbfsResponse.java`. Did you try to resolve things without adding the annotation to these first two classes in the list of files?

Reply:
That's my Jackson inexperience! Thanks for clearing it up. Works perfectly when removing the unneeded annotations!